wiki/osu!tourney/Tournament_Management_Commands/fr.md

```diff
@@ -1,3 +1,7 @@
+---
+outdated: true
+outdated_since: a28414396139e5470b683e7f02c1896659f4616f
+---
 # Commandes de gestion des tournois
```

Comment:
Please update the French article instead of outdated : 

Between L12 and L13:

```!mp makeprivate <name>- Crée une salle de tournoi privée avec le nom spécifié. Cette commande fonctionne de la même manière que `!mp make` mais l'historique des matchs n'est visible que par le créateur de la salle et le participant.```